### PR TITLE
Addresses #310 (Forms)

### PR DESCRIPTION
**Pull request by axoplasm**
This has a lot of changes to forms.py files, I would appreciate extra eyes on this before I merge.

### CODE REVIEW

File: templates/registration/bookmark_list.html [hunk 1]
```diff
@@ -1,9 +1,11 @@
 {% extends "base.html" %}
 {% block bread_crumb %}
-<ol class="breadcrumb">
-  <li><a href="accounts/profile/">Profile</a></li>
-  <li class="active">Bookmark List</li>
-</ol>
+<nav aria-label="breadcrumb">
+    <ol class="breadcrumb">
+      <li class="breadcrumb-item"><a href="accounts/profile/">Profile</a></li>
```
Comment (mahmoodkhan) [inline templates/registration/bookmark_list.html]: As you change the template files and come across hard-coded URLs, let's make an effort to update them to use  Django `{% url %}` template tag

File: templates/workflow/projectagreement_form.html [hunk 2]
```diff
@@ -7,13 +7,15 @@
     <ol class="breadcrumb">
         <li class="breadcrumb-item"><a href="/workflow/dashboard/0/">{% trans 'Projects' %}</a></li>
         {% if pk %}
-        <li class="breadcrumb-item"><a href="/workflow/dashboard/project/{{ pk }}/">Project Dashboard</a></li>
+        <li class="breadcrumb-item"><a href="/workflow/dashboard/project/{{ pk }}/">{% trans "Project Dashboard" %}</a></li>
```
Comment (mahmoodkhan) [inline templates/workflow/projectagreement_form.html]: As you change the template files and come across hard-coded URLs, let's make an effort to update them to use Django {% url %} template tag

File: templates/workflow/projectdashboard_list.html [hunk 3]
```diff
@@ -10,17 +10,25 @@
...
 
 {% block content %}
 {% if project_id == 0 %}
 
-    <div class="jumbotron">
-        <p>Name your new Project and associate it with a Program</p>
-        <form role="form" class="form-inline" action="/workflow/projectagreement_add/" method="post">
+    <div class="">
+        <p>{% trans "Name your new project and associate it with a program" %}.</p>
+        <form role="form" class="" action="/workflow/projectagreement_add/" method="post">
```
Comment (mahmoodkhan) [inline templates/workflow/projectdashboard_list.html]: As you change the template files and come across hard-coded URLs, let's make an effort to update them to use Django {% url %} template tag

**Comment by axoplasm** [comment]
I changed to named URLs in all the templates in this PR

We should put a pin in this: our URLs probably need [namespaces](https://docs.djangoproject.com/en/2.0/topics/http/urls/#url-namespaces)

**Pull request merged**